translations/translate.R

```diff
@@ -13,4 +16,312 @@ if(!exists("rootfolder"))
 
 Sys.setlocale(category = "LC_ALL", locale = "en_US.UTF-8")
 
-try(tools::update_pkg_po(rootfolder))
+
+# refactoring xgettext -> jaspXgettext
```

Comment:
Could you add comments of the form
```r
# start of changes
# end of changes
```
to each changed block of code in both functions?
See for example here: https://github.com/jasp-stats/jaspRegression/blob/3b9646176dfac865c5e96118edd53e53720af97c/R/regressionlinearbayesian.R#L814-L823

That way it'll be easier to update the code in the future.

Reply:
Sure, I will add this, thanks.